homeassistant/components/tuya/config_flow.py

```diff
@@ -98,67 +97,50 @@ def _try_login(user_input):
         _LOGGER.debug("TuyaConfigFlow._try_login finish, response:, %s", response)
         return response
 
-    async def async_step_import(self, user_input=None):
-        """Step import."""
-        self.is_import = True
-        return await self.async_step_user(user_input)
-
     async def async_step_project_type(self, user_input=None):
         """Step project type."""
         self.conf_project_type = user_input[CONF_PROJECT_TYPE]
         self.project_type = ProjectType(self.conf_project_type)
-        return (
-            self.async_show_form(step_id="user", data_schema=DATA_SCHEMA_SMART_HOME)
-            if self.project_type == ProjectType.SMART_HOME
-            else self.async_show_form(
-                step_id="user", data_schema=DATA_SCHEMA_INDUSTRY_SOLUTIONS
+        if self.project_type == ProjectType.SMART_HOME:
+            return self.async_show_form(
+                step_id="login", data_schema=DATA_SCHEMA_SMART_HOME
```

Comment:
Please don't move to another step's form directly. If we need to go to another step we need to await that step method, eg `self.async_step_login`.

A form is only allowed to be called from within the step method that has that form.

Reply:
Does this mean that only `self.async_show_form(step_id="user",data_schema=...)` can be called in `async_step_user`, but steps other than step_user cannot be called such as `self.async_show_form(step_id="project_type",data_schema=.. .)`, if I want to call `step_project_type`, I must use `await async_step_project_type` and then I can call `self.async_show_form(step_id="project_type",data_schema=...)` in async_step_project_type

Reply:
Yes. 👍